HipercowApi/Models/JobList.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Imperial College London. All rights reserved.
+
+namespace HipercowApi.Models
+{
+    using System.Diagnostics.CodeAnalysis;
+
+    /// <summary>
+    /// A list of jobs queries from the cluster.
+    /// </summary>
+    /// <param name="Jobs">
+    /// A list of JobInfo objects, giving information on each job.
+    /// </param>
+    [ExcludeFromCodeCoverage]
+    public record JobList(
```

Comment:
this probably dosent have to be its own model because its just a List<JobInfo>. but just personal preference up to you

Reply:
Agreed!

Reply:
Yeah - at some stage I think I had some meta that had to bundle with it, then decided I didn't need it yet... I'll cull that out when I've build some stuff using this listy stuff if it still seems like just a list...